### PR TITLE
[ADD] Leather thigh cuffs

### PR DESCRIPTION
**Pull request by SandrinePDR**
## References

_None_

## About The Pull Request

Added a new asset "leather thigh cuffs"

## Changelog

Authored by: Claudia & Sandrine

```md
Assets:
- [Leather thigh cuffs] New asset
```

## Checklist

- [X] The change has been tested locally
- [X] Licensing information and credits were filled in/modified for added/modified assets
- [X] I understand this change is licensed based on [Pandora's Asset Licensing](https://github.com/Project-Pandora-Game/pandora-assets/blob/master/ASSET_LICENSING.md) information

### CODE REVIEW

**Review by ClaudiaMia — CHANGES_REQUESTED** [review]
In case you were looking for all instances of washed out weirdness I found more:

image

Also how the automatic sitting transformation looks is something that needs getting used to I guess~
image

From -5 to -2 the connected chain disappears when kneeling or sitting:
image

When kneeling and sitting with 2/2 and some other values, the chain also disappears:
image

File: src/assets/foot_restraints/leather_ankle_cuffs/graphics.json [hunk 1]
```diff
@@ -425,6 +512,29 @@
...
+									"legs": "kneeling"
+								},
+								{
+									"module": "cuffState",
+									"operator": "=",
+									"value": "short"
+								},
+								{
+									"bone": "leg_l",
+									"operator": "<",
```
[the remark this answers is not in the document]
Comment (ClaudiaMia) [reply src/assets/foot_restraints/leather_ankle_cuffs/graphics.json:529]: Note that the above comment is about the ankle cuffs not the thigh ones.

**Review by Jomshir98 — CHANGES_REQUESTED** [review]
Even if it made sense, we really don't want fully-rendered and composited images in this repo.
File in question: `src/assets/foot_restraints/leather_thigh_cuffs/leatherthighcuffs_chain_kneeling_short.png`

**Comment by SandrinePDR** [comment]
You're correct. That was created by a misclick...

**Comment by ClaudiaMia** [comment]
There are still some more visual artifacts with the rings. I will (hopefully) be able to address them through further changes to https://github.com/Project-Pandora-Game/pandora-assets/pull/291 .

So for now we will have to keep this one open until I could fix these.

**Review by ClaudiaMia — APPROVED** [review]
All fixed and working.

**Review by Jomshir98 — APPROVED** [review]

**Pull request merged**